evmd/tests/network/network.go

```diff
@@ -6,6 +6,8 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	testutil2 "github.com/cosmos/evm/testutil"
```

Comment:
no more 2!!!

Reply:
fixed this and a couple more i found: https://github.com/cosmos/evm/pull/639/commits/c7747f8314cb63f1171ed90530293801a1599c7f